devops/azure-pipelines.yml

```diff
@@ -32,60 +44,54 @@ stages:
   jobs:
   - job: analysis
     steps:
-      - checkout: self
+    - checkout: self
+
+    - task: SonarCloudPrepare@1
+      inputs:
+        SonarCloud: 'SonarCloud'
+        organization: $(sonarOrg)
+        scannerMode: 'CLI'
+        configMode: 'manual'
+        cliProjectKey: $(sonarKeyPrefix)${{ variables.microservice }}
+        cliSources: './${{ variables.workDir }}'
```

Comment:
this line is a little brittle in that it requires a relative path and will break if you put an absolute path in that variable. it'd be better to not do that.

Reply:
fixed per your suggestion